Packs/Phishing/Scripts/DeleteReportedEmail/DeleteReportedEmail_test.py

```diff
@@ -261,6 +261,20 @@ def test_search_args(mocker, brand):
     current_search_args.update(ADDED_SEARCH_ARGS.get(brand, {}))
     assert get_search_args({}) == current_search_args
 
+    # Test missing message id exception
+    incident_info_copy = deepcopy(INCIDENT_INFO)
+    mocker.patch.object(demisto, 'incident', return_value=incident_info_copy)
+    incident_info_copy['CustomFields'].pop("reportedemailmessageid")
+    with pytest.raises(ValueError):
```

Comment:
why you aren't checking that you got the exact "ValueError("Message ID ('reportedemailmessageid') of the original email could not be found.")" ?

Reply:
Wanted mainly to assert there's a proper (ValueError) exception raised instead of an unexpected one later on, but I guess asserting its content as well is a good idea, I'll update it